src/ui/edittagdialog.cpp

```diff
@@ -833,27 +834,33 @@ void EditTagDialog::FetchTagSongChosen(const Song& original_song,
   const QString filename = original_song.url().toLocalFile();
 
   // Find the song with this filename
-  auto data_it =
-      std::find_if(data_.begin(), data_.end(), [&filename](const Data& d) {
-        return d.original_.url().toLocalFile() == filename;
-      });
-  if (data_it == data_.end()) {
-    qLog(Warning) << "Could not find song to filename: " << filename;
-    return;
+  int id;
```

Comment:
Is that really necessary? In the end its just a matter of personal preference but I want to produce code that is easy to understand and verifiable for everybody and does not include another new bug (which is the reason for this patch).